app/policies/apple/key_policy.rb

```diff
@@ -0,0 +1,17 @@
+class Apple::KeyPolicy < ApplicationPolicy
+  def new?
```

Comment:
Huh - out of curiousity, why wasn't this needed before?

Reply:
yeah, I forget when this popped up, but I believe it was when I was testing the Apple feed config UI...